### DISCUSSION TITLE
Error : Capstone - Data Visualization

### DISCUSSION THREAD

**Discussion by moogli90** [teamalgoritma/community#259] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107478291/183298224-63e89dfb-3933-48c7-8ca4-36da74abb621.png)
halo ijin bertanya error ini biasayany karena apa ya?

**Comment by fafilia** [comment, marked as answer]
Halo @moogli90 , izin mengkonfirmasi apakah file ini disimpan di dalam forlder yang path direktorinya terlalu dalam? Contoh:

C:\Users\username\Downloads\A\B\C\D\name_folder\file_capstone.Rmd

Jika iya seperti ilustrasi di atas, Anda boleh rename folder(dengan penamaan yang tidak terlalu panjang) atau pindahkan folder yang berisikan file Rmd ke path yang lebih general(tidak terlalu dalam, contoh:

C:\Users\username\Downloads\name_folder\file_capstone.Rmd

Jika sudah dipindahkan, silahkan restart kembali R atau RStudio Anda. Semoga bisa membantu.
